src/binary-reader-objdump.h

```diff
@@ -50,13 +50,20 @@ struct ObjdumpOptions {
   const char* section_name;
 };
 
+struct ObjdumpSymbol {
+  wabt::SymbolType kind;
+  string_view name;
```

Comment:
Maybe `std::string` here? Who owns the string_view memory?

Reply:
good point.  I was assuming it was the binary file, but I guess the strings that are provided in the reader callbacks don't outlive the reader itself.